src/virtualenv/util/__init__.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import, unicode_literals
+
+try:
+    import ConfigParser
+except ImportError:
+    # noinspection PyPep8Naming
+    import configparser as ConfigParser
```

Comment:
I tend to prefer being explicit here:

```
if sys.verison_info[0] == 2:
    import ConfigParser
else:
    import configparser as ConfigParser
```

this can prevent a class of problems where a nasty third party package provides a `ConfigParser` file in python3